spec/models/order_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe Order, type: :model do
+  let(:order){create :order}
+  let(:user){create :user}
+  subject{order}
+
+  describe "associations" do
+    context "with order_details" do
```

Comment:
không cần viết vào trong context như này đâu

Reply:
context :associations do 
  it{is_expected.to have_many(:order_details).dependent :destroy}
  it{is_expected.to belong_to :user}
end